lib/finch.ex

```diff
@@ -5,7 +5,7 @@ defmodule Finch do
              |> String.split("<!-- MDOC !-->")
              |> Enum.fetch!(1)
 
-  alias Finch.{PoolManager, Request, Response}
+  alias Finch.{PoolManager, Request, Response, SSL}
```

Comment:
This alias is unused
```suggestion
  alias Finch.{PoolManager, Request, Response}
```